docs/How-To_Guides/OpenStack/Barbican/share-secret.md

```diff
@@ -0,0 +1,41 @@
+# Sharing secrets via ACLs
+
+Normally, a Barbican secret is only available to the OpenStack API
+user that created it. However, under some circumstances it may be
+desirable to make a secret available to another user.
+
+To do so, you will need
+
```

Comment:
We should include here list of UUIDs for which users need to create ACL for. They're located on a subpage in wiki

Reply:
@noonedeadpunk  would it make sense to have _another_ page to say sharing secrets with administrators for maintenance purposes, and include the UUID of the users there?

Reply:
Is that list meant to be permanent/authoritative?

Reply:
Yes. definitive.
It can evolve over time, but it is still definitive.